modules/boost.algorithm/1.88.0/MODULE.bazel

```diff
@@ -0,0 +1,25 @@
+module(
+    name = "boost.algorithm",
+    version = "1.88.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108800,
+)
+
+bazel_dep(name = "boost.array", version = "1.88.0")
```

Comment:
Add `bazel_dep(name = "boost.pin_version", version = "1.88.0")` in all MODULE.bazel files